server/src/main/java/org/hyperskill/community/flashcards/card/ExampleDataInitializer.java

```diff
@@ -21,7 +20,7 @@
  */
 @Component
 @Slf4j
-public class ExampleDataInitializer implements CommandLineRunner {
+public class ExampleDataInitializer {
     private final MongoTemplate mongoTemplate;
 
```

Comment:
you removed the CLR? so it is not called on startup. intentionally?

Reply:
done via @PostConstruct - so fine

Reply:
`@PostConstruct` does the job.